node/src/chain_spec/testnet.rs

```diff
@@ -1,7 +1,5 @@
 use super::*;
```

Comment:
Remove this file/chainspec. I don't see the point of having 2 same chainspecs in 2 different files. We are not running this testnet anymore.

Reply:
removed